src/assets/scss/_text-sizes.scss

```diff
@@ -17,3 +17,26 @@
 .alg-h5 {
   font-size: 1.25rem;
 }
+
+.alg-text-body-1 {
+  color: var(--alg-secondary-color);
+  font-size: 1.25rem;
+  line-height: 1.2;
+  font-weight: 400;
+}
+
+.alg-text-body-2 {
```

Comment:
could we try to find a more appropriate naming than `alg-text-body-1` `alg-text-body-2` ? Does not seem practical when using it.

Reply:
Fixed